shopping-cart/register.js

```diff
@@ -0,0 +1,33 @@
+export function findProduct(cue, id) {
+
+    // loop the array
+    for (let i = 0; i < cue.length; i++) {
+        const fruit = cue[i];
+
+        // check the code against fruit.code
+        if (cue.id === id) {
+            return fruit;
+        }
+    }
+
+    // loop done
+    return null;
+}
+
+export function calcLineTotal(quantity, price) {
+    return Number((quantity * price).toFixed(2));
+}
+
+// eslint-disable-next-line no-unused-vars
+export function calcOrderTotal(id, fruits) {
```

Comment:
I see this function living in your utils.js. you probably don't need it here again.